packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.ts

```diff
@@ -736,7 +736,11 @@ export default ({ api, coreSagas, networks }) => {
       const sddEligible = yield call(api.fetchSDDEligible)
       // show SDD flow for eligible country
       if (sddEligible.eligible) {
-        return yield put(actions.components.buySell.showModal({ origin: 'WelcomeModal' }))
+        // return yield put(actions.components.buySell.showModal({ origin: 'WelcomeModal' }))
```

Comment:
remove or feature flag?

Reply:
adding a feature flag to this entire functionality